src/draw/handler/Draw.Rectangle.js

```diff
@@ -18,10 +18,17 @@ L.Draw.Rectangle = L.Draw.SimpleShape.extend({
 			fillColor: null, //same as color by default
 			fillOpacity: 0.2,
 			showArea: false,
-			showDimensions: true,
 			clickable: true
 		},
-		metric: true // Whether to use the metric measurement system or imperial
+		metric: true, // Whether to use the metric measurement system or imperial
+		precision: {
+			'ft': 2,
+			'mi': 2,
+			'nm': 2,
+			'm': 2,
+			'km': 2,
+		}, // Used in GeometryUtil.FormattedNumber
+		showDimensions: true, // Show length width in tooltip
```

Comment:
Just curious, was this in the wrong place before when it was in `shapeOption`?

Reply:
Not really. I think it made more sense since it's not shape specific. It's about the tooltip.